editor/node_3d_editor_gizmos.h

```diff
@@ -70,6 +70,23 @@ class AudioStreamPlayer3DGizmoPlugin : public EditorNode3DGizmoPlugin {
 	AudioStreamPlayer3DGizmoPlugin();
 };
 
+class Listener3DGizmoPlugin : public EditorNode3DGizmoPlugin {
+	GDCLASS(Listener3DGizmoPlugin, EditorNode3DGizmoPlugin);
+
+public:
+	bool has_gizmo(Node3D *p_spatial) override;
+	String get_gizmo_name() const override;
+	int get_priority() const override;
+
+	String get_handle_name(const EditorNode3DGizmo *p_gizmo, int p_idx) const override;
+	Variant get_handle_value(EditorNode3DGizmo *p_gizmo, int p_idx) const override;
+	void set_handle(EditorNode3DGizmo *p_gizmo, int p_idx, Camera3D *p_camera, const Point2 &p_point) override;
+	void commit_handle(EditorNode3DGizmo *p_gizmo, int p_idx, const Variant &p_restore, bool p_cancel = false) override;
```

Comment:
There's no need to override these methods if the gizmo doesn't have any handles.